src/xdocs/checks/coding/superclone.xml

```diff
@@ -34,28 +34,28 @@
         </source>
         <p id="Example1-code">Example:</p>
         <source>
-class A {
```

Comment:
How come there are no changes to `finalclass.xml` or its template?

Reply:
This is src/xdocs/checks/coding/superclone.xml why you expect change in different module finalclass.xml ?

Reply:
https://github.com/checkstyle/checkstyle/pull/13643/files#diff-9babbb3ce4f6ceeb9d9378f47fa6f39d72183a0a376d2b82166d86877ef8eb8c

Reply:
Finalclass is not templated https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/checks/design/finalclass.xml.template

Reply:
I think we can move finalclass to templates in separate PR

Reply:
Ok, thats odd we have the files Java'd already.

Reply:
https://github.com/checkstyle/checkstyle/pull/13643#discussion_r1313167293